csrankings.csv

```diff
@@ -18084,6 +18085,7 @@ Tadao Takaoka,University of Canterbury,http://www.cosc.canterbury.ac.nz/tad.taka
 Tadayoshi Kohno,University of Washington,http://homes.cs.washington.edu/~yoshi,s_YDrrgAAAAJ
 Tae Hyun Kim 0006,Hanyang University,https://sites.google.com/site/lliger9/home,NOSCHOLARPAGE
 Tae Oh,Rochester Institute of Technology,https://www.rit.edu/gccis/computingsecurity/people/tom-oh,ugbZoXwAAAAJ
+Tae-Kyun Kim,KAIST,https://sites.google.com/view/tkkim/,j2WcLecAAAAJ

```

Comment:
This person says they are at Imperial (an adjunct appointment off campus doesn't qualify for inclusion). "Associate Professor (Reader) and the director of Computer Vision and Learning Lab at Imperial College London"

Reply:
Please remove and I will merge.

Reply:
Dear Emery,
Tae-Kyun recently moved from ICL to KAIST. He is a "full-time" employee at KAIST and an "adjunct" professor at ICL. He was busy and thus was late in updating his website. Now he updated his website to reflect his changed status (https://sites.google.com/view/tkkim/). I do confirm that these two people are full-time faculties who are officially employed. Please merge these changes. Many thanks!